### ISSUE TITLE
Architecture to support Cloud Profile 5 and 6 (SCED enabled with internet inbound through SSC WAF/CAP/TIP)

### ISSUE THREAD

**Issue by CldEng-DT** [Azure/CanadaPubSecALZ#319]
Would it be possible to provide the architecture for Cloud Profile 5 and 6 to align with the usage of Express Route/SCED ?  We've onboarded SCED a while back prior to this PubSec LZ (using the Enterprise Scale LZ), but did pick a few items such as the Private DNS zones but are always hesitant since this current architecture is flagged for Cloud Profile 3 ( Cloud Only) which we are not.  Would be nice to have an entire architecture for SCED connected depts so we can realign with this model,

**Comment by ghostme** [comment]
Thank you for the feedback,
I wanted to clarify the ask here. Do you want an architecture diagram that depicts how the PubSec ALZ deployment integrates with SCED? In addition, do you want End-to-End automation which integrates with SCED's CAP/TIP resources?

**Comment by CldEng-DT** [comment]
as far as diagram, found this one in the repo : https://github.com/Azure/CanadaPubSecALZ/blob/main/docs/archetypes/hubnetwork-nva-fortigate.md but we already have a diagram created for our specific implementation.  Was looking more for End to End automation ARM/Bicep CI/CD pipelines.

**Comment by ghostme** [comment]
Thank you for the feedback.
The last mile automation/connectivity with SCED has a lot of dependencies with SSC, and it will be difficult to infuse that automation in the accelerator as we don't have insight/access to the SSC SCED environment.

**Comment by CldEng-DT** [comment]
Agree that lots of dependencies with SSC.  I'm probably looking at it more of a post "remediation" since we are already onboarded, so would be plugging in already known variables Not sure if there would be a way to set everything up but with a pre-requisite to have SCED implemented ?

**Comment by ghostme** [comment]
Thank you for the feedback. So your organization is already onboarded to SCED? are you looking at a use case where you can perhaps use some of the modules within the accelerator in your existing environment but not entirely deploy everything from the ground up?

**Comment by CldEng-DT** [comment]
Yes, we already onboarded SCED.  For our specific purpose, yes it would be nice to be able to use some modules without deploying everything so we could align to this model without potentially breaking what is currently running.  We've had to adapt the DNS/Private zone module to be able to deploy into our tenant.  Would be nice if for some of the key pieces, there could be a "Deploy to Azure" or a more generic pipeline so we can add in our DevOps instance.
